__tests__/app.test.js

```diff
@@ -20,4 +25,33 @@ describe.only("GET /api", () => {
         expect(endpoints).toEqual(endpointsJson);
       });
   });
+
+  describe("GET /api/topics", () => {
+    test("200: Responds with an array of all topics", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body }) => {
+          const { topics } = body;
+          expect(topics.length).toBeGreaterThan(0);
+          topics.forEach((topic) => {
+            expect(topic).toEqual(
+              expect.objectContaining({
+                slug: expect.any(String),
+                description: expect.any(String),
+              })
+            );
+          });
+        });
+    });
+  });
+
+  test("404: Responds with an error when the endpoint does not exist", () => {
```

Comment:
good test! Although it's not connected to any of the previous endpoints so it can have it's own describe blocka dn shouldn't be associated with any of the others :)